recipe/meta.yaml

```diff
@@ -77,7 +77,10 @@ outputs:
         - set "SSL_CERT_FILE="       # [win]
         - unset REQUESTS_CA_BUNDLE  # [unix]
         - unset SSL_CERT_FILE       # [unix]
-        - rustc --version | grep "{{ version.replace('.', '\\.') }}.*{{ year }}-{{ month }}-{{ day }}"
+        - rustc --version
+        - rustc --version 1>version.txt 2>&1
+        # the day can be off-by-one
+        - cat version.txt | grep "rustc {{ version.replace('.', '\\.') }}-nightly \(.* {{ year }}-{{ month }}-.*\)"
```

Comment:
Just out of curiosity, why does the grep test need to be split into two commands like this?

Reply:
it doesn't strictly: I was banging my head on why they didn't match until i finally realized that the day was wonky, thought it might be on stderr... It's probably still worth having the un-grepped one on line 80, but 81-83 could be combined, no doubt.

Reply:
OK. Now the rationale is documented somewhere if anyone ever wants to go back and investigate :-)